eng/build.yml

```diff
@@ -136,7 +136,7 @@ phases:
             command: push
             packagesToPush: '$(Build.SourcesDirectory)/artifacts/packages/$(_BuildConfig)/NonShipping/VS.*.nupkg'
             nuGetFeedType: external
-            publishFeedCredentials: CrossTest
+            publishFeedCredentials: 'DevDiv - VS package feed'
```

Comment:
Have you confirmed the feed endpoint linked to this service connection is the one we want?

Reply:
We have confirmed that 'DevDiv - VS package feed' maps to 'https://devdiv.pkgs.visualstudio.com/_packaging/VS/nuget/v3/index.json' which is commonly referred to as the 'VS feed'.